Hahnium.Chip8/Chip8Cpu.cs

```diff
@@ -39,45 +47,169 @@ public unsafe void Cycle()
                 case 2:
                     Op2(operands);
                     break;
-                case 10:
+                case 3:
+                    Op3(operands);
+                    break;
+                case 4:
+                    Op4(operands);
+                    break;
+                case 5:
+                    Op5(operands);
+                    break;
+                case 6:
+                    Op6(operands);
+                    break;
+                case 7:
+                    Op7(operands);
+                    break;
+                case 8:
+                    Op8(operands);
+                    break;
+                case 0xA:
                     OpA(operands);
                     break;
+                case 0xC:
+                    OpC(operands);
+                    break;
+                case 0xD:
+                    OpD(operands);
+                    break;
                 default: throw new NotImplementedException();
             }
         }
 
-        private void OpA(ushort operands)
+        private void Op0(ushort operands)
         {
-            // Sets I to the address NNN
-            this.address = operands;
+            if (operands == 0x00e0)
+            {
+                //Clears the screen.
+                Console.Clear();
+            }
+            else if (operands == 0x00ee)
+            {
+                // TODO: return
```

Comment:
Do we need separate issues open for remaining implementations?

Reply:
Maybe just bullet points in the existing Op0 issue. Most of the remaining op work is a line or two of code per unique instruction